models/get-models.js

```diff
@@ -20,4 +20,15 @@ exports.fetchedReview = (review_id) => {
         }
         return response.rows[0];
     })
+}
+
+exports.postedComment = (comment, review_id) => {
+    let data = [];
+    data.push(comment.username)
+    data.push(comment.body)
+    data.push(review_id)
+    
+    return db.query(`INSERT INTO comments (author, body, review_id) VALUES ($1, $2, $3) RETURNING *`, data).then(response => {
+        return response.rows;
```

Comment:
This response comes back later as an array, try to tweak your model to it returns just the object.